### ISSUE TITLE
StackGAN (Chapter 6) training results in noisy images

### ISSUE THREAD

**Issue by fkerem** [PacktPublishing/Generative-Adversarial-Networks-Projects#52]
Do you have any suggestions to mitigate the issue? 
We are getting very strange noisy images, no sign of any bird at all...

An example of generated image after 700 epochs:
![gen_700_2](https://user-images.githubusercontent.com/25817120/82626395-5e040700-9bf0-11ea-88c4-684c84434370.png)

**Comment by fkerem** [comment]
Here I also provide an example of a generated image:
![gen_700_2](https://user-images.githubusercontent.com/25817120/82626395-5e040700-9bf0-11ea-88c4-684c84434370.png)

**Comment by kailashahirwar** [comment]
@fkerem Please share the hyperparameters here. Also, share the loss values for the first 100 epochs.
